olm/2.10.0/openebsoperator.v2.10.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,486 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
The CSV can be updated to latest resource descriptions and correct CR spec (as per 2.10.0) in the alm example